## Description: add some description
## Origin/Author: add some origin or author
## Bug: bug URL
From 885f8275aa65fb11be1e17bc28a0b0ea734dc585 Mon Sep 17 00:00:00 2001
From: "Perry.Yuan" <953186+GoPerry@users.noreply.github.com>
Date: Wed, 20 May 2020 18:18:43 +0800
Subject: [PATCH] Revert "build_module: do not build if result will be
 obsolete"

---
 dkms | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/dkms b/dkms
index 2a93aad..15c08ef 100644
--- a/dkms
+++ b/dkms
@@ -1438,13 +1438,6 @@ sign_build()
 
 build_module()
 {
-    local buildable=0
-    for ((count=0; count < ${#built_module_name[@]}; count++)); do
-        check_version_sanity "$kernelver" "$arch" "$obsolete_by" "${dest_module_name[$count]}" || continue
-        ((buildable++))
-    done
-    ((buildable!=0)) || exit 0
-
     prepare_build
     do_build
     sign_build
